examples/testing-module/Makefile

```diff
@@ -12,44 +13,39 @@ prepare-docker:
 
 check-docker: prepare-docker
 	MODULE=docker CONFIG=./fullconfig.yaml $(CMD) $(TESTS)
-	@true
 
 check-minimal-config-docker: prepare-docker
 	MODULE=docker $(CMD) $(TESTS)
-	@true
-
 
 check-rpm: prepare-nspawn
 	MODULE=nspawn CONFIG=./fullconfig.yaml $(CMD) $(TESTS)
-	@true
 
 check-minimal-config-rpm: prepare-nspawn
 	MODULE=nspawn $(CMD) $(TESTS)
-	@true
 
 check-behave-docker: prepare-docker
 	cd ../memcached-behave; MODULE=docker behave
-	@true
 
 check-multihost-testing: prepare-nspawn
 	cd ../multios_testing; MTF_DISABLE_MODULE=yes $(CMD) *.py
-	@true
 
 check-run-them-pdc-baseruntime: prepare-nspawn
 	../../tools/run-them.sh base-runtime base-runtime-f26-20170504201340 pdc
-	@true
 
 check-run-them-pdc-testmodule: prepare-nspawn
 	../../tools/run-them.sh testmodule testmodule-master-20170413142055 pdc
-	@true
 
 check-run-them-fedmsg-testmodule: prepare-nspawn
 	../../tools/run-them.sh testmodule ../../tools/example_message_module.yaml fedmsg
-	@true
 
 check-exceptions: prepare-nspawn
-	MODULEMDURL=NOT_VALID_URL MODULE=nspawn $(CMD) --show-job-log simpleTest.py | grep 'raise ConfigExc' >/dev/null
-	@true
+	MODULEMDURL=NOT_VALID_URL  MODULE=nspawn $(CMD) --show-job-log simpleTest.py | grep 'raise ConfigExc' >/dev/null
```

Comment:
Hi @alexxa , so it would be nice when you prepare for me some example how to do it well, because I found lots of doc for these rebases on google and I've choosed one solution how to do that, so probably I choosed some worse one than your solution how to do it well.